src/modules/common-elements/dropdown.tsx

```diff
@@ -63,39 +61,35 @@ class Dropdown extends React.Component<DropdownProps, DropdownState> {
   };
 
   render() {
-    const { options } = this.props;
-    const { selected, showList, defaultStyle } = this.state;
+    const { options, large } = this.props;
+    const { selected, showList } = this.state;
     return (
       <div
-        className={defaultStyle}
+        className={large ? Styles.Dropdown_Large : Styles.Dropdown_Normal }
         ref={dropdown => {
           this.refDropdown = dropdown;
         }}
         onClick={this.toggleList}
       >
-        <button className={Styles.Dropdown__label}>{selected.label}</button>
+        <button className={Styles.Dropdown__label}>
+          {selected.label} { large ? TwoArrows : Chevron }
+        </button>
         <div
           className={classNames(Styles.Dropdown__list, {
             [`${Styles.active}`]: showList
           })}
         >
           {options.map(option => (
```

Comment:
It may be better to use a unique index here Just-in-case we have a dupe dropdown list item.

```
{options.map(option, idx) => {
  <button
    key={idx}
    value={option.value}
  >
    {option.label}
  </button>
}));